tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "suppressImplicitAnyIndexErrors": true,
     "removeComments": false,
     "sourceMap": true,
+    "lib": [ "es2015" ],
```

Comment:
Why is this necessary?

Reply:
Because of Object.assign({}, val), you can suggest something else.

Reply:
@Kiho you can replace it with `{ ...val }`. The compiler won't complain.

Reply:
@malbernaz Thank You!